docs/contributing/contributing.md

```diff
@@ -199,6 +199,17 @@ If you don't want to use [nix](https://nixos.org/guides/install-nix.html), you c
         }
       ],
       "repo": "local"
+    },
+    {
+       "repo": "https://github.com/pre-commit/pre-commit-hooks",
+       "rev": "v4.1.0",
+       "hooks": [
+          {
+            "id": "mixed-line-ending",
+            "args": ["--fix", "lf"],
+            "exclude": "test/testdata/.*CRLF*.hs$"
+          }
+       ]
```

Comment:
what about create a `.pre-commit-config.json` in the repo to make a little bit more easier and disvoerable? docs could link to the file in that case

Reply:
I can do that as well!

Reply:
Although, the pre-commit hook is part of `.gitignore`. I can envision some users use their own personal pre-commit hook and having one in source control may cause issues?